egograph/backend/tests/unit/test_validators.py

```
@@ -0,0 +1,68 @@
+    def test_nyc_zone(self):
+        """America/New_York での日付変換。"""
+        start, end = to_utc_range(
+            date(2026, 1, 15), date(2026, 1, 15), ZoneInfo("America/New_York")
+        )
+
+        # EST (UTC-5): 2026-01-15 00:00 EST = 2026-01-15 05:00 UTC
+        assert start == datetime(2026, 1, 15, 5, 0, 0)
+        assert end == datetime(2026, 1, 16, 5, 0, 0)
+
+    def test_returns_naive_datetime(self):
+        """返り値が naive datetime（tzinfo=None）であること。"""
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)